src/main/scala/tech/cryptonomic/conseil/generic/chain/DataPlatform.scala

```diff
@@ -0,0 +1,30 @@
+package tech.cryptonomic.conseil.generic.chain
+
+import tech.cryptonomic.conseil.generic.chain.DataTypes.Query
+import tech.cryptonomic.conseil.tezos.ApiOperations
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/** Companion object providing default apply implementation */
+object DataPlatform {
+  def apply(): DataPlatform = new DataPlatform(Map("tezos" -> ApiOperations))
```

Comment:
This map will eventually have to be extended. Any chance you can refer to tezos through a config rather than hardcode? Not necessarily for this pr, but eventually.

Reply:
Yes! That is my goal eventually. Similar changes I've done in PR for #69 which is currently on hold because this one has higher priority.